src/api/local_registry.ts

```diff
@@ -20,8 +20,6 @@ function getCargoPath() {
   // Trailing slash on macos (does not want / at the end) and windows (needs / at end)
   if (process.env.CARGO_HOME)
     return process.env.CARGO_HOME;
-  if (os.platform() === "win32")
-    return path.resolve(os.homedir(), "/.cargo/");
```

Comment:
https://github.com/serayuzgur/crates/commit/27553c00705cda3f4a749faaa67cc6e591663630#diff-4c576bce8593b6a1ab0040beb96a92bd99210c6ea97bf1a479b9c2fe054435b7R17

This part has been changed in the past to add `/`, but it doesn't work well, at least in my environment.

In node v14.15.1 on Windows,

```js
path.resolve("C:\\Users\UserName", "/.cargo/");
```

will output `'C:\\.cargo'`.

```js
path.resolve("C:\\Users\\UserName", ".cargo/");
```

will resolve to `'C:\\Users\\UserName\.cargo'`.

Reply:
I didn't understand, this is working for windows users (mostly it seems).
Please, if you want to help track-down the problem  and find a proper solution. Deleting the code and sending merge request is not helpfull

Reply:
In the Node.js [path.resolve](https://nodejs.org/api/path.html#path_path_resolve_paths) documentation, paths that begin with `/` are treated as absolute paths.  
On Windows, the result depends on the current directory, but an absolute path will either be a path with a drive letter (such as `C:\` or `D:\`), or in the case of a network directory, a path starting with `\\`.  
Running `path.resolve("C:\\Users\\UserName", "/.cargo/");` in the `C:\` drive and getting the result `"C:\\.cargo"` is exactly as documented and I don't understand why you added the `/`.

Even if you are not using Windows, you can run `path.win32.resolve("C:\\Users\\UserName", "/.cargo/");` to see the result on Windows. Can you give it a try?

Reply:
In order to confirm that this is not just a problem in my environment, I checked the operation with [wandbox](https://wandbox.org/permlink/1LdimUqMVNpFKHEN) and [ideone](https://ideone.com/XFZk2t) and share the results.

Reply:
Thank you.  Starting with '/' seems wrong as you said. This was the issue #82 why I added it.  Can you please check?

Reply:
The reason why `\` seems to have disappeared in #82 is that it is considered an escape character in the markdown.
As a simple example, I tried to preview the markdown 

```md
AAA\BBB  
AAA\.BBB
```
on VSCode and attached the image.

![image](https://user-images.githubusercontent.com/6465609/104718135-c3070100-576d-11eb-85b2-9ee9d1e3ec3c.png)